### DISCUSSION TITLE
Question about variables hardcoded in config/settings.py

### DISCUSSION THREAD

**Discussion by nikkwong** [nickjj/docker-flask-example#5] [General]
Hi; new to Python and trying to get adjusted to the conventions. I'm curious as to why in `config/settings.py` the values for SQLAlchemy are hardcoded:

```
pg_user = os.getenv("POSTGRES_USER", "tetricus_web_app")
pg_pass = os.getenv("POSTGRES_PASSWORD", "password")
pg_host = os.getenv("POSTGRES_HOST", "postgres")
pg_port = os.getenv("POSTGRES_PORT", "5432")
```

Is this TODO? Wouldn't the correct convention to be to use `os.getenv("POSTGRES_PASSWORD")` for example? Thank you!

**Comment by nickjj** [comment]
Hi,

The hard coded values are defaults only if you don't override them by setting the env variable.

For example chances are you'll never need to override `pg_port = os.getenv("POSTGRES_PORT", "5432")` but you can set your own `POSTGRES_PORT=1234` environment variable if you wanted to set a custom port (you wouldn't need to change this settings file).
